RecoHGCal/TICL/python/customiseTICLFromReco.py

```diff
@@ -27,12 +33,11 @@ def customiseTICLFromReco(process):
                                                 process.lcAssocByEnergyScoreProducer,
                                                 process.layerClusterCaloParticleAssociationProducer,
                                                 process.scAssocByEnergyScoreProducer,
-                                                process.layerClusterSimClusterAssociationProducer,
-                                               )
+                                                process.layerClusterSimClusterAssociationProducer)
+
     process.TICL_Validator = cms.Task(process.hgcalValidator)
     process.TICL_Validation = cms.Path(process.TICL_ValidationProducers,
-                                       process.TICL_Validator
-                                      )
+                                       process.TICL_Validator)
```

Comment:
Can you please roll back diffs that have no effect such as this line, the newline around L15, and the remove of a comma a few lines about this one?